src/pages/SosUserDetailPage.tsx

```
@@ -0,0 +1,50 @@
+import { useParams } from 'react-router-dom';
```

ESLint

[error] `react-router-dom` import should occur after import of `@_components/soshistorypage/UserInfoCard`

(import/order)